tests/integration_tests/performance/test_vsock_throughput.py

```diff
@@ -341,8 +310,7 @@ def iperf_workload(context):
 
     st_core = core.Core(name="vsock_throughput",
                         iterations=1,
-                        check_criteria=file_dumper is None,
```

Comment:
Please move this check to the `run_exercise()` call

Reply:
Done.